.rubocop_local.yml

```diff
@@ -5,6 +5,7 @@ GlobalVars:
   - $api_log
   - $aws_log
   - $azure_log
+  - $dw_log
```

Comment:
I'm all for long variable names. Why not call this `$datawarehouse_log`?
In case we want later infer the log name from the manager name, this would work out of the box too...